docs/user/automation-rules.rst

```diff
@@ -1,67 +1,87 @@
-Automation Rules
+Automation rules
 ================
 
-Automation rules allow project maintainers to automate actions on new branches and tags on repositories.
+Automation rules allow project maintainers to automate actions on new branches and tags on Git repositories.
+If you are familiar with :term:`GitOps`,
+this might seem familiar.
+The goal of Read the Docs' automation rules is to be able to control versioning through your Git repository
+and avoid duplicating these efforts on Read the Docs.
 
-Creating an automation rule
----------------------------
+.. seealso::
 
-#. Go to your project dashboard
-#. Click :guilabel:`Admin` > :guilabel:`Automation Rules`
-#. Click on :guilabel:`Add Rule`
-#. Fill in the fields
-#. Click :guilabel:`Save`
+   :doc:`/guides/automation-rules`
+     A practical guide to managing automated versioning of your documentation.
 
-How do they work?
------------------
+   :doc:`/versions`
+     General explanation of how versioning works for documentation projects on Read the Docs
+
+How automation rules work
+-------------------------
 
 When a new tag or branch is pushed to your repository,
-Read the Docs creates a new version.
+Read the Docs receives a callback to its Webhook.
+This triggers the creation of a new version.
 
-All rules are evaluated for this version, in the order they are listed.
-If the version matches the version type and the pattern in the rule,
-the specified action is performed on that version.
+All rules are evaluated for this version,
+in the order they are listed.
+If the version **matches** the version type and the pattern in the rule,
+the specified **action** is performed on that version.
```

Comment:
A diagram would be great here in the future, but probably too much for this refactor.

Reply:
If we can establish a quick way to use Mermaid diagrams, this would definitely be a valid option. Last time, I hit some issues that would need to be solved, so ended up generating Mermaid.js and saving as PNG "by hand" which was a bit too cumbersome for a "quick diagram".